src/prism-live.js

```diff
@@ -144,7 +161,16 @@ var _ = Prism.Live = class PrismLive {
 						}
 					}
 				}
-				else if (_.pairs[evt.key]) {
+				else if (this.localPairs && this.localPairs[evt.key]) {
```

Comment:
Instead of duplicating the code, assign pairs to a variable (something like `let pairs = this.localPairs || _.pairs`) and rewrite the code to use that. DRY!